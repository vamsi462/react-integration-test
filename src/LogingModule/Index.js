import { Login } from "./components/Login";
import { useLogin } from "./hooks/useLogin";

export function LoginModule() {
  const { state, onSubmit } = useLogin();
  return <Login onSubmit={onSubmit} state={state} />;
}
