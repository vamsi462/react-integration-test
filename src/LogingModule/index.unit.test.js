import { render } from "@testing-library/react";
import React from "react";
import { LoginModule } from "./Index";

describe("LoginModule", () => {
  it("matches snapshot", () => {
    const { container } = render(<LoginModule />);
    expect(container.cloneNode(true)).toMatchSnapshot();
  });
});
