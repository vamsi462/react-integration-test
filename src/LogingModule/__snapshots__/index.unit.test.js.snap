// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginModule matches snapshot 1`] = `
<div>
  <form
    autocomplete="off"
    novalidate=""
  >
    <div
      class="MuiBox-root MuiBox-root-1"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink="false"
          for="email"
          id="email-label"
        >
          Email
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input"
            id="email"
            name="email"
            type="text"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="PrivateNotchedOutline-legendLabelled-4"
            >
              <span>
                Email
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-6"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
          data-shrink="false"
          for="password"
          id="password-label"
        >
          Password
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input"
            id="password"
            name="password"
            type="password"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="PrivateNotchedOutline-legendLabelled-4"
            >
              <span>
                Password
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-7"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiButton-label"
        >
          Submit
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </form>
</div>
`;
